frontend/packages/ux-editor/src/containers/DesignView/PageAccordion/PageAccordion.test.tsx

```diff
@@ -99,6 +92,25 @@ describe('PageAccordion', () => {
     expect(mockSetSearchParams).toHaveBeenCalledWith({ layout: mockPageName2 });
   });
 
+  it('Disables delete button when isPending is true', async () => {
+    const user = userEvent.setup();
+    jest.spyOn(window, 'confirm').mockImplementation(jest.fn(() => true));
+    jest
+      .spyOn(require('./useDeleteLayout'), 'useDeleteLayout')
+      .mockImplementation(() => ({ mutate: mockDeleteFormLayout, isPending: true }));
+    await render();
+    const deleteButton = screen.getByRole('button', {
+      name: textMock('general.delete_item', { item: mockPageName1 }),
+    });
+
+    expect(deleteButton).toBeDisabled();
+    await act(async () => {
+      await user.click(deleteButton);
+    });
```

Comment:
Siden `user.click` returnerer en `Promise`, kan du bruke den direkte i stedet for å gå gjennom en `async`-funksjon som venter på den.
```suggestion
    await act(() => user.click(deleteButton));
```